.codecov.yml

```diff
@@ -11,4 +11,7 @@ coverage:
 comment:
   layout: "diff, files"
   behavior: default
-  require_changes: yes
\ No newline at end of file
+  require_changes: yes
+
+ignore:
+  - "src/vuex/modules/pushNotifications.js" 
```

Comment:
hmmmm... instead use /* istanbul: ignore */ like in the rest of the code
as well as write a test for some of the code that has logic